### PR TITLE
FAQs.md

### LINKED ISSUE #495: FAQs and Privacy Policy Statement.
Hello, I'm Harry. I'm a content creator and a copywriter and I engage in contributing to open source projects by creating documents I feel is necessary. Documents I create range from Terms of service, privacy policy to FAQs.
For your project, if it's needed by you, I could help to create FAQS and Privacy Policy for your project.
I'd love to hear your thoughts on this. Thanks

### PR DESCRIPTION
**Pull request by HarryHeights**
Here's the FAQs document I proposed to create. You can add questions you feel are missing and also proof read the document although I  already proof-read the document with grammarly.  Thanks.

# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #495

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: FAQs.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+**What is Images-to-pdf?**
+
+Images-to-pdf is an Android utility software that can convert your image files to pdf in few seconds which is the basic and primary feature of the software. The software has a lot of other useful features and they include;
+- Create PDF from multiple images from camera or gallery
+- View your converted PDFs
+- Open, Rename, Delete, print, share files
+- Sort the files order based on a number of options
+- View File details (Path, size, date created ...)
```
Comment (Swati4star) [inline FAQs.md]: Add QR code scanning code option also...

`Convert QR code or Bar code scan results to PDF`

File: FAQs.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+- The minimum password length to encrypt a pdf file is one character, however, it's not advisable to use a password with just a single numeric or alphabetic character. The software allows numeric and alphabetic password entry. For more security, create a password mixed with alphabets and numbers up to 8 characters.
+
+**I forgot my password, can I recover a lost password?**
+
+- No, a lost password cannot be recovered. While you are advised to protect your files with strong passwords which sometimes amounts to complex passwords, it is also vital it is something you can remember.
+
+**Can I change the password of a PDF file?**
+
+- This cannot be directly done but can be manipulated. To change the password, you will first have to remove the existing password from the file and create a new anew password again.
```
Comment (Swati4star) [inline FAQs.md]: A typo at the end `create a new anew password again.`

File: FAQs.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+**I'm not fluent in the English language, are there other translations of the software?**
+
+- The software is only available in English language for now. Translations to other languages would be available soon. If you feel you can contribute to this software in terms of translation feel free to send a pull request.
+
+**Does the developer have access to and collects any personal information when I use the software?**
+
+- No, the developer does not collect any information when the software is utilized because the developer does not have access to any such information. Read the [developer's privacy policy](https://sites.google.com/view/privacy-policy-image-to-pdf/home)
+
+Is Images-to-pdf software available to any other operating system?
```
Comment (Swati4star) [inline FAQs.md]: make it bold

File: FAQs.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+- There is no intention to migrate this software to another operating system.
+
+Where can I download Images-to-pdf software?
```
Comment (Swati4star) [inline FAQs.md]: make it bold

**Comment by Swati4star** [comment]
Thanks @HarryHeights 👍

**Comment by HarryHeights** [comment]
Glad I could contribute.

**Pull request merged**